src/app/pages/ParatimeDashboardPage/LearningMaterials.tsx

```diff
@@ -99,6 +100,24 @@ const getContent = (t: TFunction) => {
         },
       },
       [Layer.cipher]: undefined,
+      [Layer.pontusx]: {
+        // TODO: add proper data
```

Comment:
Why in one place runtime is hidden and here we wan't to show info ?

Reply:
So it looks like we need to adjust a type or create a new util type

Reply:
> So it looks like we need to adjust a type or create a new util type

Adjusted the type for now, so we are using `undefined` now.

We can always add some learning materials if/when we will have proper documentation.

(I will open a thread on the relevant channel about getting the docs.)